src/group_actions/group_operation_action.jl

```diff
@@ -0,0 +1,206 @@
+"""
+    LeftGroupOperation <: AbstractLeftGroupActionType
+
+A type for the [`AbstractLeftGroupActionType`](@ref) when acting on the group itself
+from the left, that is
+
+```math
+σ_h(g) = σ(h,g) = h$(_math(:∘))g
+```
+
+for its inverse ``(σ_h)^{-1}`` see [`InverseLeftGroupOperation`](@ref).
+"""
+struct LeftGroupOperation <: AbstractLeftGroupActionType end
+
+"""
+    RightGroupOperation <: AbstractRightGroupActionType
+
+A type for the [`AbstractLeftGroupActionType`](@ref) when acting on the group itself
+gfrom the right
+
+```math
+σ_h(g) = σ(h,g) = g$(_math(:∘))h
+```
+
+for its inverse ``(σ_h)^{-1}`` see [`InverseRightGroupOperation`](@ref).
+"""
+struct RightGroupOperation <: AbstractLeftGroupActionType end
+
+"""
+    InverseLeftGroupOperation <: AbstractRightGroupActionType
+
+A type for the [`AbstractLeftGroupActionType`](@ref) when acting on the group itself
+given by the inverse of a [`LeftGroupOperation`](@ref) ``σ_h`` as
+
+```math
+τ_h(g) $(_tex(:def)) σ_h^{-1}(g) = σ(h^{-1},g) = h^{-1}$(_math(:∘))g
+```
+
+Note that while in naming it is the inverse of the left action, it's
+properties yield that is is an [`AbstractRightGroupActionType`](@ref), since
+
+$(_note(:LeftInverseActionIsRight))
+
+for its inverse ``(σ_h)^{-1}`` see [`InverseLeftGroupOperation`](@ref).
+
+!!! note
+   Some literature also calls this by itself _the_ right group operation action.
+"""
+struct InverseLeftGroupOperation <: AbstractRightGroupActionType end
+
+"""
+    InverseRightGroupOperation <: AbstractLeftGroupActionType
+
+
+A type for the [`AbstractLeftGroupActionType`](@ref) when acting on the group itself
```

Comment:
```suggestion
A type for the [`AbstractLeftGroupActionType`](@ref) when acting on the group itself
```
Unnecessary empty line